jib-maven-plugin/README.md

```diff
@@ -306,15 +315,37 @@ In this configuration, the image:
 
 You can add arbitrary, non-classpath files to the image by placing them in a `src/main/jib` directory. This will copy all files within the `jib` folder to the image's root directory, maintaining the same structure (e.g. if you have a text file at `src/main/jib/dir/hello.txt`, then your image will contain `/dir/hello.txt` after being built with Jib).
 
-You can configure a different directory by using the `extraDirectory` parameter in your `pom.xml`:
+You can configure a different directory by using the `extraDirectory` parameter in your `pom.xml`, either by using the `<extraDirectory><path>` parameter:
 
 ```xml
-<configuration>
-  ...
-  <!-- Copies files from 'src/main/custom-extra-dir' instead of 'src/main/jib' -->
-  <extraDirectory>${project.basedir}/src/main/custom-extra-dir</extraDirectory>
-  ...
-</configuration>
+<!-- Copies files from 'src/main/custom-extra-dir' instead of 'src/main/jib' -->
+<extraDirectory>
+  <path>${project.basedir}/src/main/custom-extra-dir</path>
+</extraDirectory>
+```
+
+or by setting the path on the `extraDirectory` parameter directly:
+
+```xml
+<!-- Copies files from 'src/main/custom-extra-dir' instead of 'src/main/jib' -->
+<extraDirectory>${project.basedir}/src/main/custom-extra-dir</extraDirectory>
+```
+
+The `extraDirectory` parameter can also be used to set the file permissions of files added to the container by specifying the absolute path of the file on the container as well as the octal representation of its file permission bits in the `<extraDirectory><permissions>` field:
+
+```xml
+<extraDirectory>
```

Comment:
Similarly, when using `<permissions>`, it is still possible to set the path without `<path> like below?

```xml
<extraDirectory>
   /some/path
  <permissions>...</permissions>
</extraDirectory>
```